src/dlgTriggerEditor.cpp

```diff
@@ -90,38 +90,63 @@ dlgTriggerEditor::dlgTriggerEditor(Host* pH)
+    msgInfoAddButton = tr("<p>Buttons react on mouse clicks. To add a new button:"
+                          "<ol><li>Add a new group to define a new button bar in case you don't have any.</li>"
+                          "<li>Add new groups as menus to a button bar or sub-menus to menus.<li>"
+                          "<li>Add new items as buttons to a button bar or menu or sub-menu.</li>"
+                          "<li>Define a clear text command that you want to send to the game if the button is pressed, or write a script for more complicated needs.</li>"
+                          "<li><strong>Activate</strong> the toolbar, menu or button. <strong>Note:</strong> Deactivated items will be hidden and if they are toolbars or menus then all the items they contain will be also be hidden.</li></ol></p>"
+                          "<p><strong>Note:</strong> If a button is made a <strong>click-down</strong> button then you may also define a clear text command that you want to send to the game when the button is pressed a second time to uncheck it or to write a script to run when it happens - within such a script the Lua 'getButtonState()' function reports whether the button is up or down.</p>"
                           "<p>Check the manual for <a href='http://wiki.mudlet.org/w/Manual:Contents'>more information</a>.</p>");
 
-    msgInfoAddKey = tr("<p>To add a new key binding:"
-                       "<ul><li><b>1.</b> Add a new key</li>"
-                       "<li><b>2.</b> Click on <b><i>grab key</i></b> and then press your key combination.<b> "
-                       "<u>NOTE:</u></b> If you want to bind a key combination you must hold down the modifier keys (e.g. control, shift etc.) down before clicking on grab key.</li>"
-                       "<li><b>3.</b> Define a command that is executed when the key is hit.</li>"
-                       "<li><b>4. <u>Activate</u></b> the new key binding.</li></ul></p>"
+    msgInfoAddKey = tr("<p>Keys react on keyboard presses. To add a new key binding:"
+                       "<ol><li>Click on the 'Add Item' icon above.</li>"
+                       "<li>Click on <strong>'grab key'</strong> and then press your key combination."
+                       "<strong>NOTE:</strong> If you want to bind a key combination you must hold down the modifier keys (e.g. Control, Shift, etc.) down before clicking on 'grab key'.</li>"
+                       "<li>Define a command that is executed when the key is hit.</li>"
+                       "<li><strong>Activate</strong> the new key binding.</li></ol></p>"
+                       "<p><strong>Note:</strong> Keys can also be defined from the command line in the main profile window like this:</p>"
+                       "<p><code>lua tempKey(mudlet.key.F8, function() send(&quot;jump&quot;) end)</code> - Pressing F8 will make you jump.</p>"
```

Comment:
:bulb: It is probably better to use `permkey` - that will show up in the editor and given that only the first key-binding will fire (unless the option is set) the user can get into difficulties if they forget the id number of the temporary one(s) as they can't delete ones they create with the same key binding and only the first one will fire even if they rerun the command line and replace the `function()` with a more advanced one as they try and develop it...

Reply:
Yeah I was pondering that, but decided against, as the API is a bit more complex.
Reviewing it now, your arguments do hold more weight. I'll just leave *parent* empty.